src/services/tableDataServices.js

```diff
@@ -9,3 +9,13 @@ export const signUp = async (body) => {
     return e;
   }
 };
+
+export const signIn = async (body) => {
+  try {
+    const response = await request("/api/users/log-in", "POST", body);
+    console.log(response);
+    return response;
```

Comment:
and send only required data it is causing a lot of checks on client like the first comment

Reply:
if i am sending require data from here... it will crash my sign in process so I am dispatching only required data t store